### PR TITLE
OSDOCS-8816 z-stream RNs for 4.11.54

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.11

Issue: [OSDOCS-8816](https://issues.redhat.com/browse/OSDOCS-8816)

Link to docs [preview:](https://68437--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes#ocp-4-11-54)

QE review:
- [ ] QE has approved this change.
Not needed

Additional information: Expected merge date is Nov 29th. Links will not work until then.

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -3843,3 +3843,37 @@ $ oc adm release info 4.11.53 --pullspecs
...
+$ oc adm release info 4.11.54 --pullspecs
+----
+
+[id="ocp-4-11-54-features"]
+==== Features
+
+[id="ocp-4-11-54-APIserver-tracked-by-Insights-Operator"]
+===== APIServer.config.openshift.io is now tracked by Insights Operator
+
+After running the Insights Operator, a new file is now available in the archive in the path `config/apiserver.json` with the information about the audit profile.
```
Comment (dfitzmau) [inline release_notes/ocp-4-11-release-notes.adoc:3867]: The plus sign is showing in the preview. I think you can merge the sentences into one paragraph as not much text exists. Do you have a link to a Jira for this feature?

Consider removing the two running `in the` clauses to something such as ` is now available as an archive file in the `config/apiserver.json` path. The archive file contains information about the audit profile.`

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -3843,3 +3843,37 @@ $ oc adm release info 4.11.53 --pullspecs
...
++
+Access to audit profiles allows you to understand what audit policy is common practice, what profiles are most commonly used, what differences there are between industries, and what kind of customisation is applied.
```
Comment (dfitzmau) [inline release_notes/ocp-4-11-release-notes.adoc:3869]: "Allows" is a form of anthropomorphism (snippet from IBM SG):

Screenshot from 2023-11-29 10-41-35

How about:

> By accessing audit profiles, you can determine what audit policy meets common practice, commonly adopted profiles, differences that exist between profiles adopted by industries, and levels of customisation for profiles.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -3843,3 +3843,37 @@ $ oc adm release info 4.11.53 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.11.54 --pullspecs
+----
+
+[id="ocp-4-11-54-features"]
+==== Features
+
+[id="ocp-4-11-54-APIserver-tracked-by-Insights-Operator"]
```
Comment (dfitzmau) [inline release_notes/ocp-4-11-release-notes.adoc:3864]: The [OCP guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata) doc states to lower case anchor IDs. 

ocp-4-11-54-APIserver-tracked-by-Insights-Operator/ ocp-4-11-54-apiserver-tracked-by-insights-operator

**Comment by dfitzmau** [comment]
Hi @cbippley . I added some feedback. I've never used the Features heading for a z-stream so this is interesting. Would you be OK to move the Jira to the "Review" stage?

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (cbippley) [reply release_notes/ocp-4-11-release-notes.adoc:3869]: Good catch!

Comment (cbippley) [reply release_notes/ocp-4-11-release-notes.adoc:3869]: I found the same feature in a 4.13 z release so I'm altering this to match it.

**Review by jldohmann — APPROVED** [review]
LGTM, nice work Courtney 😄

**Pull request merged**